exposed-tests/src/test/kotlin/org/jetbrains/exposed/sql/tests/shared/ddl/CreateIndexTests.kt

```diff
@@ -83,4 +82,68 @@ class CreateIndexTests : DatabaseTestsBase() {
             assertEquals(true, TestTable.exists())
         }
     }
+
+
+    @Test
+    fun `test partial index`() {
+        val partialIndexTable = object : IntIdTable("PartialIndexTableTest") {
+            val name = varchar("name", 50)
+            val value = integer("value")
+            val anotherName = integer("anotherName")
+            val anotherValue = integer("anotherValue")
+            val flag = bool("flag")
+            init {
+                index("flag_index", columns = arrayOf(flag, name)) {
+                    flag eq true
+                }
+                index(columns = arrayOf(value, name)) {
+                    (name eq "aaa") and (value greaterEq 6)
+                }
+                uniqueIndex(columns = arrayOf(anotherValue))
+            }
+        }
+
+        withDb(TestDB.POSTGRESQL) {
```

Comment:
Honestly, no idea.

Reply:
I think we should, because even though `PostgreSQLNGDialect` extends `PostgreSQLDialect`, in TestDB, they are separate.

Reply:
@lure The line you added [here](https://github.com/JetBrains/Exposed/pull/1748/files#:~:text=return%20if%20(currentDialect%20is%20PostgreSQLDialect)%20%7B) means that the changes you introduced will apply to both `PostgreSQLDialect` _and_ `PostgreSQLNGDialect`, since `PostgreSQLNGDialect` extends `PostgreSQLDialect`, which means tests need to cover both. Thanks for adding it in the test 👍🏼